### PR TITLE
add default filter config

### PR DESCRIPTION
**Pull request by ansyral**

### CODE REVIEW

**Comment by chenkennt** [comment]
:shipit:

**Comment by hellosnow** [comment]
:shipit:

File: src/Microsoft.DocAsCode.Metadata.ManagedReference/Visitors/SymbolVisitorAdapter.cs [hunk 1]
```diff
@@ -893,6 +895,39 @@ private NamedArgumentInfo GetNamedArgumentInfo(KeyValuePair<string, TypedConstan
...
+                defaultRule = YamlUtility.Deserialize<ConfigFilterRule>(reader);
+            }
+
+            if (string.IsNullOrEmpty(filterConfigFile))
+            {
+                return defaultRule;
+            }
+            else
+            {
+                userRule = YamlUtility.Deserialize<ConfigFilterRule>(filterConfigFile);
```
Comment (qinezh) [inline src/Microsoft.DocAsCode.Metadata.ManagedReference/Visitors/SymbolVisitorAdapter.cs]: Do we need to handle the possible exception?

**Comment by superyyrrzz** [comment]
:shipit:

**Comment by superyyrrzz** [comment]
:+1:

File: src/Microsoft.DocAsCode.Metadata.ManagedReference/Microsoft.DocAsCode.Metadata.ManagedReference.csproj [hunk 2]
```diff
@@ -107,6 +107,9 @@
     <Compile Include="Models\MetadataModel.cs" />
   </ItemGroup>
   <ItemGroup>
+    <EmbeddedResource Include="Filters\defaultfilterconfig.yml">
+      <LogicalName>$(AssemblyName).Filters.defaultfilterconfig.yml</LogicalName>
+    </EmbeddedResource>
```
Comment (vwxyzh) [inline src/Microsoft.DocAsCode.Metadata.ManagedReference/Microsoft.DocAsCode.Metadata.ManagedReference.csproj]: and embedded resource in `project.json`.

File: src/Microsoft.DocAsCode.Metadata.ManagedReference/Visitors/SymbolVisitorAdapter.cs (hunk 1 above)
Comment (ansyral) [inline src/Microsoft.DocAsCode.Metadata.ManagedReference/Visitors/SymbolVisitorAdapter.cs]: i think it is better that the exception handler code put in YamlUtility wrapper. for this case, i don't want to swallow the user error if user wrote illegal yaml config file

**Comment by qinezh** [comment]
:+1:

**Pull request merged**